app/routes/news/show.js

```diff
@@ -6,7 +6,13 @@ export default Route.extend({
   prismic: service(),
 
   model({ uid }) {
-    return this.prismic.getNewsItemByUid(uid);  
+    return this.prismic.getNewsItemByUid(uid);
```

Comment:
Je pense qu'il serait intéressant d'expliciter le fait que `const posts = this.prismic.getNewsItemByUid(uid);`. Mais du coup, `newsItemByUid`, est-ce que ça constitue vraiment un `post` ?

Reply:
Post n'est pas forcement le bon, j'opte pour `model` et du coup je ne trouve pas utile de faire un `const model`